files/config/scheduler.yaml

```diff
@@ -1,8 +1,4 @@
-kind: KubeSchedulerConfiguration
-algorithmSource:
-  provider: DefaultProvider
 apiVersion: kubescheduler.config.k8s.io/v1alpha1
+kind: KubeSchedulerConfiguration
 clientConnection:
   kubeconfig: /etc/kubernetes/kubeconfig/scheduler.yaml
-failureDomains: kubernetes.io/hostname,failure-domain.beta.kubernetes.io/zone,failure-domain.beta.kubernetes.io/region
-hardPodAffinitySymmetricWeight: 1
```

Comment:
Moved to scheduler policy config in https://github.com/kubernetes/kubernetes/pull/44159